active/PipsPager/PipsPager.md

```diff
@@ -0,0 +1,324 @@
+| PipsPagerDefaultIndicatorFontSize | Sets the size in pixels of the pip glyph | Double |
+| PipsPagerSelectedIndicatorFontSize | Sets the size in pixels of the pip glyph when selected | Double |
+| PipsPagerDefaultForegroundBrush | Sets the foreground brush of the default pip glyphs| SolidColorBrush
+| PipsPagerSelectedForegroundBrush | Sets the foreground brush of the pip glyph when selected | SolidColorBrush
+| PipsPagerPointerOver | Sets the pointer over/hover visual state for the pip display mode | Brush
+| PipsPagerPressed | Sets the pressed visual state for the pip display mode | Brush
+| PipsPagerSelected | Sets the selected visual state for the pip display mode | Brush
+| PipsPagerMargin | Sets the margin, in pixels, between each pip glyph | Thickness
+| PipsPagerIndicatorWidth | Sets the bounding box width for each pip glyph | Double
+| PipsPagerIndicatorHeight | Sets the bounding box height for each pip glyph | Double
+
+
+## PipsPager member notes
+
+| Name | Description| Default | 
+|:---:|:---|:---|
+| NumberOfPages | Sets the max number of pages the index control will iterate through. The default will represent an infinite page range. | -1
+| Previous and Next Style | Give the developer the option to customize the style by changing the text or glyph for the edge buttons. | N/A
+| SelectedPageIndex | The 0 based index that is currently selected. It will default to the first index. | 0
+| MaxVisiblePips | The maximum number of pips that will appear in the control at once. If the NumberOfPages is greater than this value, the UI will scroll so that the selected pip is centered. | 5
+| Orientation | The orientation of the control. Can be either Vertical or Horizontal. | Horizontal
+| PipsPagerSelectedIndexChangedEvent | Event that is fired after the user selects a pip or the directional buttons. This event will return the index number of the page that the user selected. | N/A
+
+## PipsPagerButtonVisibility enum
+
+Defines constants that specify how a the buttons of a PipsPager are to be displayed. Defaults to Collapsed.
+
+| Value | Description |
+|:---:|:---|
+| Visible | The button is visible and selectable. An exception to this is that if the selected page is on the associated "edge" of the button then the button will be hidden |
+| VisibleOnPointerOver | The behavior is the same as Visible *except* that the button is only Visible when the user is hovering over the paging UI with their cursor. |
+| Collapsed | The button is not visible to the user and does **not** take up layout space. |
+
+
+# API Details
+
+```IDL
+enum PipsPagerButtonVisibility
+{
+  Visible,
+  VisibleOnPointerOver,
+  Collapsed, 
+}
+
+runtimeclass PipsPagerSelectedIndexChangedEventArgs
+{
+    Int32 NewPageIndex{get; };
+    Int32 PreviousPageIndex{get; };
+};
+
+unsealed runtimeclass PipsPagerTemplateSettings : Windows.UI.Xaml.DependencyObject
+{
+    Windows.Foundation.Collections.IVector<Object> PipsPagerItems { get; }; 
```

Comment:
@StephenLPeters @beervoley Could this be int ? Is the content always ints?

Reply:
@ranjeshj it's only ints. This array is just to keep the number of pips for the Repeater to keep.

Reply:
I'll leave this open for now and we can discuss in review. The value of the item in PipsPagerItems isn't required or used -- only the fact that an item is there to use to calculate the size of the items collection.

Reply:
Discussion: it can be anything, currently it's an int. Making this object-typed means that those ints are getting boxed though, which is unnecessary overhead.

Recommend: IVector<Int32>